opencti-platform/opencti-front/src/private/components/import/Import.jsx

```diff
@@ -1,27 +1,32 @@
 import React from 'react';
 import * as PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom-v5-compat';
 import { QueryRenderer } from '../../../relay/environment';
 import ImportContent, { importContentQuery } from './ImportContent';
 import Loader from '../../../components/Loader';
 
-const Import = () => (
-  <QueryRenderer
-    query={importContentQuery}
-    variables={{}}
-    render={({ props }) => {
-      if (props) {
-        return (
-          <ImportContent
-            connectorsImport={props.connectorsForImport}
-            importFiles={props.importFiles}
-            pendingFiles={props.pendingFiles}
-          />
-        );
-      }
-      return <Loader />;
-    }}
-  />
-);
+const Import = () => {
```

Comment:
This component should be untouched

Reply:
Why ?

Reply:
This way, this component does not bear a useless concern